trajopt_sco/include/trajopt_sco/osqp_interface.hpp

```diff
@@ -0,0 +1,59 @@
+#pragma once
+#include <Eigen/Core>
+#include <osqp.h>
+
+#include <trajopt_sco/solver_interface.hpp>
+#include <trajopt_utils/macros.h>
+
+namespace sco
+{
+class OSQPModel : public Model
+{
+  OSQPSettings osqp_settings_;     // OSQP Settings
+  OSQPData osqp_data_;             // OSQPData
+  OSQPWorkspace* osqp_workspace_;  // OSQP Workspace
+
+  void updateObjective();
+  void updateConstraints();
+  void createOrUpdateSolver();
+
+public:
+  VarVector vars_;
```

Comment:
Documentation for member variables would be helpful for other developers in the future.

Reply:
Actually in many cases these variables could safely be private. Should we document them also in those cases?

Reply:
I think would be used for debugging or adding features later on for those not familiar.